.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
   - TEST_GROUP="cucumber -f progress -r features features/other_b"
   - TEST_GROUP="cucumber -f progress -r features features/prompt_memes_a"
   - TEST_GROUP="cucumber -f progress -r features features/prompt_memes_b"
+  - TEST_GROUP="cucumber -f progress -r features features/prompt_memes_c"
```

Comment:
I couldn't get the test to pass which they were in two files, spliting them up got the tests to pass so that is why it was done.